CHANGELOG.md

```diff
@@ -0,0 +1,274 @@
+
+### Installer Binaries
+| filename | Size | sha512 hash |
+| -------- | ---- | ----------- |
+| [keadm-v0.3.0-linux-amd64.tar.gz](https://github.com/kubeedge/kubeedge/releases/download/v0.3.0/keadm-v0.3.0-linux-amd64.tar.gz) | 2.23 MB | `bce839fef66e0e370876dbbdc671129f3e209d03e7530992ea84bb0bc804fd0a88182db06852266a3cde80e52c0b8ae7bc8f58ff2022acb88f67dd8d6f5b11d3` |
+
+
+# KubeEdge v0.3 Release Notes
+
+## 0.3 What's New
+
+**Device Management using CRD**
+
+This feature mainly provides APIs for managing devices from cloud and synchronize the device updates between cloud and edge.
+DeviceController is the module in cloud which is responsible for watching on devices and devicemodels and syncing the data to edge. DeviceController is also responsible for receiving the device-twin updates sent from edge and making the changes in cloud(apiserver). DeviceController also creates a configmap for each node and adds the deviceInstances, protocols, properties and propertyVisitors in it. This configmap can be consumed by any application like mapper and used to configure/control devices.
+
+**Bluetooth Mapper**
+
+Bluetooth Mapper is an application that is used to connect and control devices that use bluetooth as a communication protocol. It consumes the configmap generated by device-controller and reuires a configuration file for information not present in configmap. Configurations can be changed at runtime by providing the input through the MQTT broker. It can run schedule based actions and converts data received from devices into edge_core expected format.
+
+**Performance Test Framework**
+
+Benchmarking framework to measure the performance against the following Service Level Objectives:
+    - Latency: time cost from the moment when the server gets the request to last byte of response sent to the users.
+    - Throughput: measure how many requests can be served within given time.
+    - Scalability: potential scaling capacity (including number of nodes, pods, devices etc. at the edge) under different load conditions.
+    - CPU Usage: measure the cpu usage of KubeEdge under different load conditions.
+    - Memory Usage: measure the memory usage of KubeEdge under different load conditions.
+
+**KubeEdge Installer**
+Support for basic commands to bootstrap and teardown both KubeEdge cloud and edge components.
+Supported Commands :
+    - `kubeedge init` : bootstrap cloud prerequisites and cloud components if not already present.
+    - `kubeedge join` : add edge node in cloud api-server, bootstrap edge prerequisites and edge components if not already present.
+    - `kubeedge reset` : for cloud node teardown cloud components and prerequites. For edge node teardown edge components and prerequisites.
```

Comment:
prerequites -> prerequisites

Reply:
this description looks repetitive : 
for cloud node teardown cloud components and prerequites. For edge node teardown edge components and prerequisites.

this can be simplified to : 
teardown cloud and edge components and associated prerequisites.

Reply:
Done, Thanks

Reply:
I was think that "teardown cloud and edge components and associated prerequisites" gives an idea that running kubeedge reset on a node will teardown cloud and edge components both. @rohitsardesai83 , can you suggest something that will make it clear that we need to run it on both cloud and edge nodes ?

Reply:
tears down the installed components/prerequisites. The reset action has to be done on the cloud and edge nodes separately to ensure proper cleanup.

Reply:
Done, Thanks